pkg/datagatherer/k8s/dynamic.go

```diff
@@ -420,17 +442,17 @@ func namespaceResourceInterface(iface dynamic.NamespaceableResourceInterface, na
 	return iface.Namespace(namespace)
 }
 
-// generateFieldSelector creates a field selector string from a list of
-// namespaces to exclude.
-func generateFieldSelector(excludeNamespaces []string) string {
-	fieldSelector := fields.Nothing()
```

Comment:
I think there was a bug in this function. This initial `fields.Nothing()` caused the returned selector to always have a trailing comma which was ugly when I appended the user-supplied field selectors.